modules/apps/object/object-test/src/testFunctional/tests/ObjectNotifications.testcase

```diff
@@ -560,6 +560,93 @@ definition {
 		}
 	}
 
+	@description = "LPS-178784 - Verify that it's possible to send an e-mail notification with Action (Download)"
+	@priority = 5
+	test CanSendEmailViaActionDownload {
+		property custom.properties = "feature.flag.LPS-173894=true";
```

Comment:
This property is not more necessary, the behavior is already on Master.

Reply:
unnecessary FF removed